core/base/inc/TStorage.h

```diff
@@ -88,7 +88,23 @@ class TStorage {
    static Bool_t IsOnHeap(void *p);
 
    static Bool_t FilledByObjectAlloc(volatile const UInt_t* const member);
-   static void UpdateIsOnHeap(volatile const UInt_t &uniqueID, volatile UInt_t &bits);
```

Comment:
Question, does using 'just' also silence the warning (or does it fail to compile)?
```
< static void UpdateIsOnHeap(volatile const UInt_t &uniqueID, volatile UInt_t &bits);
> static R__NEVER_INLINE void UpdateIsOnHeap(volatile const UInt_t &uniqueID, volatile UInt_t &bits);